invenio_rdm_records/services/pids/providers/external.py

```diff
@@ -41,9 +41,10 @@ def __call__(self, record, identifier, provider, errors):
         for p in self.prefixes:
             if identifier.startswith(p):
                 errors.append(
-                    _("The prefix '{prefix}' is administrated locally.").format(
-                        prefix=p
+                    _("The prefix '{prefix}' is managed by {sitename}.").format(
+                        prefix=p, sitename=current_app.config["THEME_SITENAME"]
                     )
+                    + " Please supply an external DOI or select 'No' to have a DOI generated for you."
```

Comment:
This needs to be included inside the `_()` so it can be extracted for translation.

```python 
_("The prefix '{prefix}' is managed by {sitename}. Please supply...", prefix=..., sitename=...).```